### ISSUE TITLE
Terminal rendered not correct - missing last line

### ISSUE THREAD

**Issue by DrDeath** [GitSquared/edex-ui#393]
On a mac, the Terminal is cutting the last line in all themes.

Cutted last line:
![edex-ui_terminal](https://user-images.githubusercontent.com/198929/51181796-69447580-18cc-11e9-8ae7-5924a4fbe22f.png)

Normal last line in a Terminal Session:
![terminal](https://user-images.githubusercontent.com/198929/51181809-73667400-18cc-11e9-8530-6fd99290a3dd.png)

**Comment by GitSquared** [comment]
Does this still happen if you make a new tab?

**Comment by DrDeath** [comment]
Yes, it happens on all Tabs...

![edex-ui-terminal-2](https://user-images.githubusercontent.com/198929/51182336-1ec3f880-18ce-11e9-88ee-303019daad6d.png)

**Comment by VonGrav** [comment]
Greetings, Lovely project this one. 
I suffer the same problem and it comes from running on this resolution for me: 
2560x1440. 
If I switch to 1920x1080 the issue does not appear.

**Comment by GitSquared** [comment]
Yes, we had this bug previously - see #302 - and it's related to screen resolution. I [implemented a fix](https://github.com/GitSquared/edex-ui/blob/master/src/classes/terminal.class.js#L158) that corrects the automatic sizing of the terminal depending on the screen ratio, but it seems that it's actually more related to specific width and height somehow...

@DrDeath could you please tell me your monitor resolution?

and @VonGrav thanks for your help, you did pretty much figure it out ;)

**Comment by DrDeath** [comment]
of course, I'm using an 27-Inch iMac at 2560 x 1440 Pixel

**Comment by GitSquared** [comment]
@DrDeath All right, thanks for your help! I'm afraid i don't have enough time to work on open-source much during the week but i'll make sure to submit a bunch of bugfixes next weekend in a v2.0.1 release.

**Comment by DrDeath** [comment]
never mind, take your time :-)

**Comment by Grizaster** [comment]
Same here! Screen resolution of 2560 x 1440 with Windows 10.
Just for your information! No pressure ;)
